crates/library/std/src/buf.fe

```diff
@@ -0,0 +1,43 @@
+use ingot::evm
+
+unsafe fn avail() -> u256 {
```

Comment:
We should probably expose that as an unsafe function somewhere outside of the memory buffer.

Reply:
Yep, I was expecting it to probably go away in the future with the rewrite. I was just thinking about the *current* Fe because this function may end up being duplicated across a bunch of different places in the std library.

Reply:
I was thinking we should only use functions in a single lower level type and then swap out the internals when v2 lands

Reply:
I guess I'm just generally more ok to break user code while we are in `0.x` land. As I see it, users shouldn't expect API stability in the current phase of the project. 😉

Reply:
From my perspective, this is not a simple API stability problem.